### ISSUE TITLE
Anonymize repository id taken

### ISSUE THREAD

**Issue by lucala** [tdurieux/anonymous_github#273]
No matter what I enter, I always get the error that the repository id has been taken. This seems very strange...

**Comment by tdurieux** [comment]
Are you sure you don't try to reuse the id of a repo that you removed?

**Comment by lucala** [comment]
yes, you're right. This was the problem, I wanted to rename my repo and then generate a new url.

**Comment by mhellmeier** [comment]
But there is still a bug regarding the _repository id is already used_ error.

## Steps to reproduce

1. Visit the `/anonymize` page to anonymize a repository
2. Use a repository id that is already taken
3. Click the "Anonymize" button
4. An error occurs that the repository id is already taken (this error is correct)
5. Change the repository id to something that is not used
6. The error remains there, even if the repository id is unused

## Example
Example Screenshot of step 6:
![image](https://github.com/tdurieux/anonymous_github/assets/1036284/18dc1f7e-ffc0-4508-a480-7d77869ad233)

## Quick fix
After step 6, click the "Anonymize" button again. This will work.

**Comment by loooffeeeey** [comment]
Can I reuse the id that have been removed by myself?
